cinema/serializers.py

```diff
@@ -114,10 +125,10 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
```

Comment:
The `validate` method in `TicketSerializer` could be improved by checking if the required keys ('row', 'seat', 'movie_session') are present in `attrs` before accessing them. This would prevent potential `KeyError` exceptions if any of these keys are missing.